src/main/java/com/valmiki/adhyapakservice/dao/CoursesRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.valmiki.adhyapakservice.dao;
+
+import com.valmiki.adhyapakservice.entity.Courses;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CoursesRepository extends JpaRepository<Courses, Integer> {
```

Comment:
rename dao package to repo or repository.

Reply:
fixed